src/OpenAI.csproj

```diff
@@ -46,6 +46,6 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" PrivateAssets="All" />
     <PackageReference Include="System.ClientModel" Version="1.1.0-beta.4" />
-    <PackageReference Include="System.Text.Json" Version="8.0.2" />
+    <PackageReference Include="System.Memory.Data" Version="9.0.0-preview.4.24266.19" />
```

Comment:
Yes, strange, I checked with 8.0.0, but maybe some problems with auto-restore in the IDE when I did this and a transitive dependency on System.ClientModel was used
![image](https://github.com/openai/openai-dotnet/assets/3002068/87b84761-8e30-4a28-ae15-d0976984d168)

P.S. I think System.ClientModel should update this

Reply:
> P.S. I think System.ClientModel should update this

Yeah :/  System.ClientModel won't be able to update this beyond 6.x for right now because of its dependency on STJ. It would break the Azure Functions host. But I wish too